Content.Client/HUD/GameHud.cs

```diff
@@ -428,6 +430,12 @@ public void Initialize()
             LC.SetGrowVertical(VoteContainer, LC.GrowDirection.End);
         }
 
+        private void OpenInfoWindow()
```

Comment:
```suggestion
        private void OpenRulesWindow()
```

Reply:
So, the class is named InfoWindow. InfoWindow contains the rules tab and a simple help tab.